client/components/payment-activity/test/index.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import PaymentActivity from '..';
+
+describe( 'PaymentActivity component', () => {
+	it( 'should render', () => {
+		const { getByText, container } = render( <PaymentActivity /> );
+
+		expect( getByText( 'Your payment activity' ) ).toBeInTheDocument();
+		expect(
```

Comment:
What do think about adding a snapshot test here for the component?

Reply:
Thanks for pointing this out! Testing is among my new areas. I read about snapshots and pushed this change:

https://github.com/Automattic/woocommerce-payments/pull/8412/commits/100652868001228bdc34bbbf85e68542fefe4c9c

I hope this is done correctly?